proxy-node-translator/src/test/java/uk/gov/ida/notification/translator/saml/HubResponseTranslatorTest.java

```diff
@@ -27,14 +29,16 @@
         }
     }
 
+    private Supplier<EidasResponseBuilder> eidasResponseBuilderSupplier = () -> EidasResponseBuilder.instance();
```

Comment:
Try changing `() -> EidasResponseBuilder.instance()` to  `EidasResponseBuilder::instance`

Reply:
Applied this suggestion